author	Johan Hedberg <johan.hedberg@intel.com>
commit	e61df1600a09334beab122a9580161ae1fabdca5
	Wed, 18 Apr 2012 11:17:51 +0000 (14:17 +0300)

audio: Fix AVDTP Reconfigure Reject message

The specification describes a proper reject response to
AVDTP_Reconfigure, so the previously used unknown command response is
not correct.

diff -cr bluez-4.99.pan.orig/audio/avdtp.c bluez-4.99.pan/audio/avdtp.c
*** bluez-4.99.pan.orig/audio/avdtp.c	Fri Oct 12 08:21:05 2012
--- bluez-4.99.pan/audio/avdtp.c	Mon Oct 15 05:10:37 2012
***************
*** 1634,1640 ****
  static gboolean avdtp_reconf_cmd(struct avdtp *session, uint8_t transaction,
  					struct seid_req *req, int size)
  {
! 	return avdtp_unknown_cmd(session, transaction, AVDTP_RECONFIGURE);
  }
  
  static gboolean avdtp_open_cmd(struct avdtp *session, uint8_t transaction,
--- 1634,1646 ----
  static gboolean avdtp_reconf_cmd(struct avdtp *session, uint8_t transaction,
  					struct seid_req *req, int size)
  {
! 	struct conf_rej rej;
! 
! 	rej.error = AVDTP_NOT_SUPPORTED_COMMAND;
! 	rej.category = 0x00;
! 
! 	return avdtp_send(session, transaction, AVDTP_MSG_TYPE_REJECT,
! 					AVDTP_RECONFIGURE, &rej, sizeof(rej));
  }
  
  static gboolean avdtp_open_cmd(struct avdtp *session, uint8_t transaction,
